### PR TITLE
fix typo, mention case of wrong riverbank

### PR DESCRIPTION
**Pull request by matkoniecz**
Is there sane method to update `po/` files? Or is it necessary to do it manually?

### CODE REVIEW

**Comment by frodrigo** [comment]
The english version has to be fixed in the code.

**Comment by matkoniecz** [comment]
> in the code.

Which one? Is my PR sufficient or should I edit also something else?

File: analysers/analyser_osmosis_waterway.py [hunk 1]
```diff
@@ -149,8 +149,8 @@ def __init__(self, config, logger = None):
 '''There is one `waterway=riverbank` but there is no `waterway=river`
 inside it.'''),
             fix = T_(
-'''After checking, create a "river" line inside te "riverbank"
-polygon.'''))
+'''After checking, create a "river" line inside the "riverbank"
+polygon or eliminate "riverbank" polygon.'''))
```
Comment (frodrigo) [inline analysers/analyser_osmosis_waterway.py:153]:
```suggestion
polygon or eliminate then "riverbank" polygon.'''))
```
```suggestion
polygon or eliminate "riverbank" polygon.'''))
```

**Comment by frodrigo** [comment]
> > in the code.
> 
> Which one? Is my PR sufficient or should I edit also something else?

Yes, sorry I did not see at first time it is a PR and not an issue.

File: analysers/analyser_osmosis_waterway.py (hunk 1 above)
Comment (matkoniecz) [reply analysers/analyser_osmosis_waterway.py:153]: I am unsure what is being proposed here. Adding word "then" and removing it?

Comment (frodrigo) [reply analysers/analyser_osmosis_waterway.py:153]: Ho, it is a typo. I was suggesting adding "the".

**Pull request merged**